frontend/src/lib/constants.tsx

```diff
@@ -135,11 +135,12 @@ export const FEATURE_FLAGS = {
     YULE_HOG: 'yule-hog', // owner: @benjackwhite
     QUERY_RUNNING_TIME: 'query_running_time', // owner: @mariusandra
     DATA_EXPLORATION_INSIGHTS: 'data-exploration-insights', // owner @thmsobrmlr
+    DATA_EXPLORATION_QUERIES_ON_DASHBOARDS: 'data-exploration-queries-on-dashboards', // owner: @pauldambra
+    DATA_EXPLORATION_QUERY_TAB: 'data-exploration-query-tab', // owner: @pauldambra
```

Comment:
Is there a big difference between these? Basically, where to add users who want access to this? Both?